.github/stale.yml

```diff
@@ -1,21 +1,18 @@
 # Configuration for probot-stale - https://github.com/probot/stale
 
 # Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 700
+daysUntilStale: 186
 
 # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
 # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-daysUntilClose: 60
+daysUntilClose: 21
```

Comment:
Why 21? I think marking until "stale" is fine, but lets give more time for someone when they got a notification from bot. I think 60 was a reasonable amount for a complex project such as Rizin is.